ccd-definition/CaseEvent.json

```diff
@@ -156,7 +156,7 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "UNSPECIFIED_CLAIMS",
     "ID": "NOTIFY_DEFENDANT_SOLICITOR_FOR_CLAIM_ISSUE",
-    "Name": "Notify defendant solicitor",
+    "Name": "Notify claim issue",
```

Comment:
I'm not sure we can be as generic as this, I think we will need a different event for each email

Reply:
Might be a good idea to have them separated in ccd. Do you also think that Camunda should be sending list of ccd events to trigger in single service task, or would it be better to trigger each one separately?

Reply:
I believe we've said it will be indivudal tasks so we can accurately know what has and hasn't happened on a case. @ckalista will be able to inform you better here I am sure :)

Reply:
Yeah it needs to be a seperate Service Task per CCD Event or else how would we recover if say event II in the list failed in CCD...?

Reply:
we agreed in past that we will have same External task listener used with Case_Event name hard coded in parameters of specific Camunda Task